### DISCUSSION TITLE
Error: spawn prisma-client.js ENOENT

### DISCUSSION THREAD

**Discussion by anand-dehru** [prisma/prisma#20883] [Q&A] [answered]
### Question

I am facing this issue

**Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Error: spawn prisma-client.js ENOENT**

this is my package.json file
```json
{
  "name": "prisma",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "nodemon"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "@prisma/client": "^5.2.0",
    "dotenv": "^16.3.1",
    "express": "^4.18.2",
    "mongodb": "^6.0.0",
    "nodemon": "^3.0.1",
    "prisma": "^5.2.0"
  }
}
```

### Information about Prisma Schema, Client Queries and Environment (optional)

and it is my schema.prisma file
```prisma
generator client {
    provider = "prisma-client.js"
}

datasource db {
    provider = "mongodb"
    url      = env("DATABASE_URL")
}

model User {
    id        String   @id @default(auto()) @map("_id") @db.ObjectId
    firstName String
    lastName  String
    age       Int
    createdAt DateTime @default(now())
    updatedAt DateTime @updatedAt
}
```

**Comment by ludralph** [comment, marked as answer]
Hi @anand-netzoptimize 👋 

Thank you for raising this question.

In your generator block, you have `prisma-client.js` as the provider value. Can you please change it to the below:

```prisma
generator client {
  provider = "prisma-client-js"
}
```

Notice that there is no dot in the name `prisma-client-js`.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
